config/config.go

```diff
@@ -1454,3 +1457,15 @@ func ContainHiddenConfig(s string) bool {
 	}
 	return false
 }
+
+// GetGlobalKeyspaceName is used to get global keyspace name
+// from config file or environment variable.
+func GetGlobalKeyspaceName() string {
+	// The cfg.keyspaceName get higher priority than KEYSPACE_NAME in system env.
+	keyspaceName := GetGlobalConfig().KeyspaceName
+	if keyspaceName != "" {
+		return keyspaceName
+	}
+
+	return os.Getenv(EnvVarKeyspaceName)
```

Comment:
It's better to avoid this.
Environment is not easy to control.

Reply:
Can we add this to the config or command line?

Reply:
Ok, the env setting can be removed.